Index: app/src/main/java/live/wellconnect/wellconnect/data/DataRepositoryImpl.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/live/wellconnect/wellconnect/data/DataRepositoryImpl.kt b/app/src/main/java/live/wellconnect/wellconnect/data/DataRepositoryImpl.kt
--- a/app/src/main/java/live/wellconnect/wellconnect/data/DataRepositoryImpl.kt	
+++ b/app/src/main/java/live/wellconnect/wellconnect/data/DataRepositoryImpl.kt	
@@ -53,15 +53,14 @@
                     "email" to user.email,
                     "password" to user.password,
                     "name" to user.name,
-                    //"photo" to getImage(user.photo),
                 )
             )
     }
 
     private fun getImage(photo : String) : Task<Uri> {
-        var file = Uri.fromFile(File("$photo"))
+        val file = Uri.fromFile(File("$photo"))
         val imagesRef = referenceStorage.child("images/${file.lastPathSegment}")
-        var uploadTask = imagesRef.putFile(file)
+        val uploadTask = imagesRef.putFile(file)
 
         // observers de la imagen subida
         uploadTask.addOnFailureListener {
